Framework/Kernel/src/Material.cpp

```diff
@@ -72,8 +115,22 @@ double Material::pressure() const { return m_pressure; }
  * the given wavelength
  */
 double Material::cohScatterXSection(const double lambda) const {
-  (void)lambda;
-  return m_element.coh_scatt_xs;
+  UNUSED_ARG(lambda);
```

Comment:
Could the lack of caching impact performance for the absorption correction algorithms?